### DISCUSSION TITLE
College email option not listed in verification

### DISCUSSION THREAD

**Discussion by bhoffman0** [community/community#60341] [GitHub Education]
### Hi everyone,

I'm a faculty member at College of our Lady of the Elms (Elms College) trying to get verified. On your system, you only accept the `name@student.elms.edu` email addresses, which is correct for students at our college, but not for faculty. Faculty and Staff have the email addresses `name@elms.edu`, without the `students` subdomain. Could you please fix your system to accept any email address ending in `elms.edu`? Thanks. -Beryl
<img width="688" alt="image" src="https://github.com/community/community/assets/4571962/2b9aa25b-e082-4cf6-a6d7-3b0ef01897d7">

**Comment by samus-aran** [comment]
Hi @bhoffman0  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
